include_pdf_latex

zsigmas opened the pull request:
The `pdflatex` package is not found in the new image (in fact, the package is installed but after autoremoved). Without it, manuals cannot be checked when `rcmdcheck` runs.

- In the previous image it was also not installed, nonetheless checks run fine without it.
- Unclear why in this new set of versions it becomes a requirement
- No relevant documentation or information was found regarding this discrepancy.

Proposed Solution:
To resolve the issue, explicitly install the `pdflatex` package in the new image. Unclear if further investigation about the origin of the discrepancy will bring any extra benefit.

zsigmas committed:
include_pdf_latex

ml-ebs-ext commented on scripts/install_sys_deps.sh:
```diff
@@ -8,6 +8,7 @@ lbzip2 \
 wget \
+texlive-latex-base\
 "
 apt-get update -y
```
Maybe we could write down the reason for the inclusion of this system dependency: 
```suggestion
# texlive-latex-base because it contains `pdflatex`
apt-get update -y
```

If we were to do it for each of them (I'm not suggesting we do it now), the bash array syntax would allow to inline the reasons:
```
# List of system dependencies
pkgs_to_install=(
  curl
  lbzip2
  rsync
  qpdf
  wget
  texlive-latex-base # contains pdflatex
)

apt-get update -y
# shellcheck disable=SC2086
apt-get install -q -y ${pkgs_to_install[@]}
```

ml-ebs-ext reviewed (approved):
Agree with the analysis and with the "maybe not getting to the bottom of this right now".

zsigmas committed:
include comment for texlive-latex-base

It is the package that includes pdflatex, but it is not obvious based on the package name

Co-authored-by: ml-ebs-ext <157474668+ml-ebs-ext@users.noreply.github.com>
